_opensearch/query-dsl/query-string.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: Query string queries
+parent: Query DSL
+nav_order: 70
+---
+
+# Query string queries
+
+A `query_string` query parses the query string based on the `query_string` syntax. 
+
+## Example
+
+The following query searches for the speaker `KING` in the play name that ends with `well`:
+
+```json
+GET shakespeare/_search
+{
+ "query": {
+    "query_string": {
+      "query": "speaker:KING  AND play_name: *well"
+    }
+  }
+}
+```
+
+## Parameters
+
+Query string query supports the following parameters. All parameters except `query` are optional.
+
+Parameter | Data type | Description
+:--- | :--- | :---
+`query` | String | The query string to use for search. Required.
+`fields` | String array | The list of fields to search (for example, `"fields": ["title^4", "description"]`). Supports wildcards.
+`default_field` | String | The field in which to search if the field is not specified in the query string. Supports wildcards. Defaults to the value specified in the `index.query.default_field` index setting. By default, the `index.query.default_field` is `*`, which means extract all fields eligible for term query and filter the metadata fields. The extracted fields are combined into a query if the `prefix` is not specified. Eligible fields do not include nested documents. Searching all eligible fields could be a resource-intensive operation. The `indices.query.bool.max_clause_count` search setting defines the maximum value for the product of the number of fields and the number of terms that can be queried at one time. The default value for `indices.query.bool.max_clause_count` is 4,096.
+`allow_leading_wildcard` | Boolean | Specifies whether `*` and `?` are allowed as first characters of a search term. Default is `true`.
+`analyze_wildcard` | Boolean | Specifies whether OpenSearch should attempt to analyze wildcard terms. Some analyzers do a poor job at this task, so the default is `false`.
```

Comment:
Are we saying that some of OpenSearch analyzers do a poor job? Do we need to qualify why the default is false? Suggestion: Delete the qualifier so the sentence reads "The default is `false`."

Reply:
Could not find any justification as to why the default is false, so removing the justification.